sample-apps/react/messenger-clone/src/components/QuickDial/QuickDial.tsx

```diff
@@ -26,6 +21,7 @@ const useGetUsers = () => {
             // FIXME: find proper solution for filtering online users
             { online: true },
           ],
+          // @ts-expect-error not available in stream-chat
           id: { $nin: [client.user!.id] },
```

Comment:
this bad boy should be removed :) We could probably use the UserSearchSource here. Putting it here for me to do the change once the PR is merged